guides/common/modules/proc_importing-a-repository.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Importing_a_Repository_{context}"]
+= Importing a Repository
```

Comment:
Please rename to "Importing a Yum Repository", including the anchor and file name.

Reply:
@maximiliankolb Does "Yum Repository" embrace all the exportable repo types, i.e. RPM, Kickstart and file repositories?

Reply:
OK. If we keep it generic, then please also ensure that it covers DEB content :) See line 4 and line 7. Please ignore the comment for "exporting-a-repository.adoc" but also ensure if covers DEB content.

Reply:
Got it. Good point!